assets/src/edit-story/components/library/panes/pageTemplates/templateList.js

```diff
@@ -76,6 +86,18 @@ function TemplateList({ pages, parentRef, pageSize, handleDelete, ...rest }) {
     overscan: 4,
   });
 
+  useEffect(() => {
+    if (
+      rowVirtualizer.virtualItems.length &&
+      rowsTotal &&
+      rowsTotal - THRESHOLD <
+        rowVirtualizer.virtualItems[rowVirtualizer.virtualItems.length - 1]
+          .index
+    ) {
+      fetchTemplates?.();
```

Comment:
When would it not be defined?

Reply:
For the default templates there's no fetching extra currently, it's a static list.

Reply:
Ah yes, I realized that later. Thanks!